sites/docs/src/content/docs/guidelines/components/modules.md

```diff
@@ -120,13 +120,14 @@ E.g. in the first example, `bwa mem` is the first tool so is given `$args`, `sam
 ### Types of meta fields
 
 Modules MUST NOT use 'custom' hardcoded `meta` fields.
+This means both that they should not be referred to within the module as expected input, nor generate new fields as output.
 The only accepted 'standard' meta fields are `meta.id` or `meta.single_end`.
 Proposals for other 'standard' fields for other disciplines must be discussed with the maintainers team.
```

Comment:
should we provide a location for these proposals? e.g. modules channel?

Reply:
I guess we could...